parcels/particlefile/baseparticlefile.py

```diff
@@ -17,7 +17,7 @@
 try:
     from parcels._version import version as parcels_version
 except:
-    raise EnvironmentError('Parcels version can not be retrieved. Have you run ''python setup.py install''?')
+    raise OSError('Parcels version can not be retrieved. Have you run ''python setup.py install''?')
```

Comment:
Does EnvironmentError not exist anymore? This really is an issue with the Environment!

Reply:
See my other https://github.com/OceanParcels/parcels/pull/1328#discussion_r1127736973